vllm/attention/ops/blocksparse_attention/blocksparse_attention_kernel.py

```diff
@@ -3,6 +3,407 @@
 import triton.language as tl
 
 
+@triton.jit
+def _context_fwd_kernel_inner(
```

Comment:
It's better to write done the math formulation implemented by this kernel. The same for other kernel functions.